docs/technical_reference/unit_models/electrolyzer.rst

```diff
@@ -0,0 +1,245 @@
+   "molar generation of species according the cathode electrolysis reaction", ":math:`\dot{n}_{cat,j} = \varepsilon_{cat,j}\dot{n}_{e^-}`"
+   "reversible voltage", ":math:`V_{rev} = E_{ano}-E_{cat}`"
+   "cell voltage", ":math:`V_{cell} = V_{rev}+\eta_{ano}+\eta_{cat}+IR`"
+   "power", ":math:`P = IV_{cell}`"
+   "electrons contributing to reactions :math:`^c`", ":math:`\dot{n}_{e^-} = \frac{I\theta_{current}}{F}`"
+   "voltage efficiency", ":math:`V_{rev} = V\theta_{voltage}`"
+   "power efficiency", ":math:`\theta_{power} = \theta_{current}\theta_{voltage}`"
+
+\ :math:`^c` is Faraday's constant from ``idaes.core.util.constants``, 96,485 C/mol
+
+Costing Method
+---------------
+
+Costing Method Variables
++++++++++++++++++++++++++
+
+The following parameters are constructed when applying the electrolyzer costing method in the ``watertap_costing_package``:
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Default Value", "Units"
+
+   "membrane replacement factor (fraction of membrane replaced/year)", ":math:`f_{memreplace}`", "factor_membrane_replacement", "0.33", ":math:`\text{dimensionless}`"
+   "membrane unit cost", ":math:`c_{membrane}`", "membrane_unit_cost", "25", ":math:`\frac{$}{m^2}`"
+   "anode unit cost", ":math:`c_{anode}`", "anode_unit_cost", "300", ":math:`\frac{$}{m^2}`"
+   "cathode unit cost", ":math:`c_{cathode}`", "cathode_unit_cost", "600", ":math:`\frac{$}{m^2}`"
+   "membrane, anode, and cathode fraction of total capital", ":math:`f_{material}`", "fraction_material_cost", "0.65", ":math:`\text{dimensionless}`"
+
+The following variables are constructed when applying the electrolyzer costing method in the ``watertap_costing_package``:
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Units"
+
+   "membrane capital cost", ":math:`C_{membrane}`", "membrane_cost", ":math:`$`"
+   "anode capital cost", ":math:`C_{anode}`", "anode_cost", ":math:`$`"
+   "cathode capital cost", ":math:`C_{cathode}`", "cathode_cost", ":math:`$`"
+   "membrane replacement cost", ":math:`C_{memreplace}`", "membrane_replacement_cost", ":math:`\frac{$}{yr}`"
+
+Capital Cost Calculations
++++++++++++++++++++++++++
+
+Capital costs are contributing to the majority of material costs for the anode cathode and membrane. Each material cost is calculated individually then summed (O’Brien, 2005).
+
+    .. math::
+
+        & C_{membrane} = c_{membrane}A_{membrane} \\\\
+        & C_{anode} = c_{anode}A_{anode} \\\\
+        & C_{cathode} = c_{cathode}A_{cathode} \\\\
+        & C_{cap,total} = \frac{C_{membrane}+C_{anode}+C_{cathode}}{f_{memreplace}}
+
+Operating Cost Calculations
++++++++++++++++++++++++++++
+
+Operating costs for the electrolyzer are the electricity requirements and membrane replacement costs. Electricity is costed using ``cost_flow`` applied to the ``power`` variable on the unit model. Currently, replacement costs for the anode and cathode are not considered in the costing function.
```

Comment:
Thinking about this a little more, if we cost the flow by power, which I presume is in `kW`, would this be accurate if we were to encounter a non-hourly timescale? What I mean is, at an hourly temporal resolution, power (kW) and energy (kWh) would match, but that wouldn't hold true at other temporal scales.